lib/system/basic_types.nim

```diff
@@ -20,10 +20,6 @@ const
   off* = false  ## Alias for ``false``.
 
 type
-  Ordinal*[T] {.magic: Ordinal.} ## Generic ordinal type. Includes integer,
-                                 ## bool, character, and enumeration types
-                                 ## as well as their subtypes.
-
```

Comment:
Why did you move this declaration to `system.nim`?

Reply:
it's an include file so doesn't make a difference; I moved it to fix bootstrap; once we upgrade nim version in csources to 1.0.X, we can put it back there